recipes/vibration.py

```diff
@@ -63,7 +67,7 @@ def __init__(self, molecule, settings, jobType=ADFJob, get_gradients='get_gradie
         if reorder != None:
             self.reorder = getattr(self.jobType._result_type, reorder)
         else:
-            self.reorder = lambda x: x
+            self.reorder = _dummy_reorder
```

Comment:
I'm a bit confused as to what exactly is going here.
`reorder` has a default value of `"inputOrder"`, yet I cannot find any `Results.inputOrder` attribute anywhere in PLAMS.

Secondly, why does `_dummy_reorder` have two parameters all of a suden?

Reply:
I checked this upon your comment. I've had this there because the ADF interface used to not have the gradients in the input order (I guess the RKFs did not have it back then). So this extra step was needed. But from what I see now, all the interfaces now by default use the correct input order, so this is indeed outdated.
I'll remove the whole thing, it is anyway better if the user overwrites the get_gradients method if needed.

And later on in the code, `self.reorder` is called with two arguments, that is why the dummy takes two. It would just fail with lambda...